doc/source/ray-air/examples/analyze_tuning_results.ipynb

```diff
@@ -164,13 +164,14 @@
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "id": "228ae052",
    "metadata": {},
    "source": [
     "With these parameters in place, we'll create a Ray AIR `XGBoostTrainer`.\n",
     "\n",
-    "Note a few things here. First, we pass in a `scaling_config` to configure the distributed training behavior of each individual XGBoost training job. Here, we want to distribute training across 2 workers.\n",
+    "Note a few things here. First, we pass in a `scaling_config` to configure the distributed training behavior of each individual XGBoost training job. Here, we want to distribute training across 2 workers. We also make sure to keep some CPU resources free for Ray Data operations.\n",
```

Comment:
```suggestion
    "Note that we pass in a `scaling_config` to configure the distributed training behavior of each individual XGBoost training job. We want to distribute training across 2 workers. We also keep some CPU resources free for Ray Data operations.\n",
```

Reply:
this suggestion should go into master as well! @angelinalg